frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/status-card.tsx

```diff
@@ -39,7 +39,7 @@ export const CephAlerts = withDashboardResources(
     return (
       <AlertsBody error={!_.isEmpty(loadError)}>
         {loaded &&
-          alerts.length &&
+          alerts.length > 0 &&
```

Comment:
Does it solve the zero error?

Reply:
Yes, because jsx does not render boolean values.